packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1036,3 +1037,10 @@ export const MAX_FRAMESIZES = {
   [QUALITY_LEVELS.MEDIUM]: 3600,
   [QUALITY_LEVELS.HIGH]: 8192
 };
+
+
+/*
+*  mqa Interval for sending stats metrics
+*/
+
+export const MQA_INTEVAL = 60000; // mqa analyzer interval its fixed to 60000
```

Comment:
Can we add this as a config value instead?

Reply:
Its a fixed value and user cannot config it